Cargo.toml

```diff
@@ -36,20 +37,25 @@ features = ["minwindef", "ws2def", "winerror"]
 [dev-dependencies]
 env_logger = "0.6.0"
 crossbeam-channel = "0.3.6"
-generic-channel = { version = "0.2.0", features = ["all"] }
 systemstat = "0.1.3"
-ping = { path = "protocols/ping", package = "tentacle-ping" }
-identify = { path = "protocols/identify", package = "tentacle-identify" }
-discovery = { path = "protocols/discovery", package="tentacle-discovery" }
 
 [target.'cfg(unix)'.dev-dependencies]
 nix = "0.13.0"
 
+[features]
+default = []
```

Comment:
Why not choose one as the default?

Reply:
I think we should have no default tendency, more is user choice.

and with default, dependence must write by:
```toml
p2p = { path = "../..", version = "0.2.0", package = "tentacle", default-features = false, features = ["molc"] }
```

but if there is no default choose:
```
p2p = { path = "../..", version = "0.2.0", package = "tentacle",  features = ["molc"] }
```

Reply:
I think we should have no default tendency, more is user choice.

and with default, dependence must write by:
```toml
p2p = { path = "../..", version = "0.2.0", package = "tentacle", default-features = false, features = ["molc"] }
```

but if there is no default choose:
```
p2p = { path = "../..", version = "0.2.0", package = "tentacle",  features = ["molc"] }
```

Reply:
I think we should have no default tendency, more is user choice.

and with default, dependence must write by:
```toml
p2p = { path = "../..", version = "0.2.0", package = "tentacle", default-features = false, features = ["molc"] }
```

but if there is no default choose:
```
p2p = { path = "../..", version = "0.2.0", package = "tentacle",  features = ["molc"] }
```

Reply:
Maybe we need to add such kind of code snippet to make it fail fast?

```
#[cfg(all(not(feature = "flatc"), not(feature = "molc")))]
compile_error!("Please choose a serialization format via feature. Possible choices: flatc, molc");
```